### ISSUE TITLE
Getting decoding output <unk> <unk> <unk> <unk>

### ISSUE THREAD

**Issue by Teenuu** [wenet-e2e/wenet#982]
I got the correct decoding results with pretrained models with LM and with out LM. But with my acoustic model decoding results are <unk> and nothing with out LM and with LM respectively. I have trained conformer model using my data with librispeech recipie. 

010010007 ▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>
010010041 ▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>
010010064 ▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>
010010093 ▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>
010010100 ▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>
010010103 ▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>
010010144 ▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>▁<unk>

**Comment by robin1001** [comment]
What is loss during your training?

**Comment by Teenuu** [comment]
> 

The loss for first epoch itself is 3.26.

**Comment by maituan** [comment]
hello, i have same problem when trying with a subset 100h of common voice dateset, have you solved yet?

**Comment by xingchensong** [comment]
u may refer to

- https://github.com/wenet-e2e/wenet/issues/825 
- https://github.com/wenet-e2e/wenet/issues/519
- https://github.com/wenet-e2e/wenet/issues/1673

**Comment by xingchensong** [comment]
and 
- https://github.com/wenet-e2e/wenet/issues/1545
